app/src/main/res/values-es/strings.xml

```diff
@@ -211,8 +222,10 @@
 
     <string name="connectionWizard_providerSelection_titleMessage">Selecciona un proveedor de wallabag</string>
     <string name="connectionWizard_providerSelection_provider_general_name">Otro</string>
-    <string name="connectionWizard_providerSelection_provider_general_desc">Selecciona esta opción para introducir detalles manualmente</string>
+    <string name="connectionWizard_providerSelection_provider_v2wallabagorg_name" translatable="false">v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_general_desc">Selecciona esta opción para introducir los ajustes requeridos manualmente</string>
     <string name="connectionWizard_providerSelection_provider_v2wallabagorg_desc">Selecciona esta opción si tienes una cuenta en https://v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_framabag_name" translatable="false">Framabag.org</string>
```

Comment:
Does anybody know if we really need the strings which are `translatable="false"` in every language file? I would remove them, too. But i do not know if this has any impact if those strings are only available in the values/strings.xml